### PR TITLE
return more related labels

### PR DESCRIPTION
**Pull request by jamesgorrie**
For travel particularly.

### CODE REVIEW

**Comment by theefer** [comment]
:+1:

File: media-api/app/lib/elasticsearch/ElasticSearch.scala [hunk 1]
```diff
@@ -125,7 +125,8 @@ object ElasticSearch extends ElasticSearchClient with SearchFilters with ImageFi
       }
   }
 
-  def labelSiblingsSearch(structuredQuery: List[Condition], excludeLabels: List[String] = Nil)(implicit ex: ExecutionContext): Future[AggregateSearchResults] = {
+  def labelSiblingsSearch(structuredQuery: List[Condition], excludeLabels: List[String] = Nil, size: Int = 20)
```
Comment (jamesgorrie) [inline media-api/app/lib/elasticsearch/ElasticSearch.scala]: I've added it as a param here, but don't want to open up the API to have a infinite number (like the rest of our API :scream:)

Comment (theefer) [inline media-api/app/lib/elasticsearch/ElasticSearch.scala]: Elsewhere we should enforce a maximum.

Comment (jamesgorrie) [inline media-api/app/lib/elasticsearch/ElasticSearch.scala]: Indeed.

**Pull request merged**